draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -101,13 +98,13 @@
   <middle>
     <section title="Introduction" anchor="Introduction">
       <t>
-        This specification describes how a CBOR Web Token <xref target="CWT"/> can declare
+        This specification describes how a CBOR Web Token <xref target="RFC8392"/> can declare
```

Comment:
Please update this from `CBOR Web Token <xref target="RFC8392"/>` to `CBOR Web Token (CWT) <xref target="RFC8392"/>` so that we're using the full spec name and keeping the abbreviation.